src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Link, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { fetchRegistration } from '../../../api/registrationApi';
+import { Registration, RegistrationStatus } from '../../../types/registration.types';
+
+interface DeletedRegistrationInformationProps {
+  registration: Registration;
+}
+
+export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+  const { t } = useTranslation();
+  const duplicateRegistrationQuery = useQuery({
+    enabled: !!registration.duplicateOf,
+    queryKey: ['registration', registration.duplicateOf],
+    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    meta: { errorMessage: t('feedback.error.get_registration') },
+  });
+
+  const duplicateRegistrationTitle =
+    duplicateRegistrationQuery.data?.entityDescription?.mainTitle ?? registration.duplicateOf;
+  const unpublishingNote = registration.publicationNotes?.find((note) => note.type === 'UnpublishingNote');
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.1rem',
+        marginTop: '1rem',
+      }}>
+      <Box
+        sx={{
+          bgcolor: 'publishingRequest.main',
+          display: 'grid',
+          gridTemplateColumns: '1fr auto',
+          alignItems: 'center'
+          gap: '0.5rem 1rem',
+          padding: '0.2rem 1rem',
+          borderRadius: '4px',
+        }}>
+        {(registration.status === RegistrationStatus.Unpublished ||
+          registration.status === RegistrationStatus.Deleted) && (
+          <Typography variant="h4" component="span" sx={{ textTransform: 'uppercase' }}>
+            {t(`registration.status.${registration.status}`)}
+          </Typography>
+        )}
+
+        {unpublishingNote?.note && <Typography>{unpublishingNote.note}</Typography>}
+        {registration.duplicateOf && (
+          <Typography>
+            {t('registration.citation_points_to')}{' '}
+            <Link href={registration.duplicateOf}>{duplicateRegistrationTitle}</Link>
+          </Typography>
+        )}
+      </Box>
```

Comment:
Ah seff, ser nå at dette egentlig er en remake av visningen av publiseringsevents som vi allerede har. Bedre at alle events ser like ut enn at de er implementer litt ulikt på ulike plasser. Å fikse design for dette er da igjen en egen oppgave som noen må prioritere, men når vi prioriterer det vet vi at de vil fikses likt overalt. Ellers kan jeg nesten garantere at de blir avvikende 😅 

(forslaget er bare en copy-paste fra det vi har i `PublishingAccordion`, så mulig det må justeres litt)

```suggestion
<StyledStatusMessageBox sx={{ bgcolor: 'publishingRequest.main' }}>
  <Typography>{t(`registration.status.${registration.status}`)}</Typography>
  {registration.publishedDate && (
    <Typography>{new Date(registration.publishedDate).toLocaleDateString()}</Typography>
  )}

  {unpublishingNote?.note && <Typography sx={{ gridColumn: '1/3' }}>{unpublishingNote.note}</Typography>}
  {registration.duplicateOf && (
    <Typography sx={{ gridColumn: '1/3' }}>
      {t('registration.citation_points_to')}{' '}
      <Link href={registration.duplicateOf}>{duplicateRegistrationTitle}</Link>
    </Typography>
  )}
</StyledStatusMessageBox>
```